.pre-commit-config.yaml

```diff
@@ -18,4 +18,4 @@ repos:
     -   id: flake8
         exclude: (^src/sadie/anarci/aa/|notebooks/)
         additional_dependencies: [flake8-bugbear]
-        args: ["--ignore=W503,E501,E203,B006,B008"]
+        args: ["--ignore=W503,E501,E203,B006,B008,B019"]
```

Comment:
cache through a possible "memory leak" issue that was already fixed. ignore it with B019